base/cvd/cuttlefish/host/commands/cvd/cli/parser/load_config.proto

```diff
@@ -18,6 +18,12 @@ syntax = "proto3";
 
 package cuttlefish.cvd.config;
 
+/*
+ * TODO: If this proto changes, then the generated proto code should too.
+ * We should check they are kept in sync.
+ */
+option go_package = "github.com/google/android_cuttlefish/base/cvd/cuttlefish/host/commands/cvd/cli/parser/golang/";
```

Comment:
let's add documentation on how the existing `base/cvd/cuttlefish/host/commands/cvd/cli/parser/golang/load_config.pb.go` is generated using the new `load_config_go_proto` bazel rule

Reply:
Done, see base/cvd/cuttlefish/host/commands/cvd/cli/parser/BUILD.bazel.